src/pay/batch_payer.py

```diff
@@ -160,19 +166,31 @@ def pay(self, payment_items_in, dry_run=None):
             if pi.needs_activation and self.delegator_pays_ra_fee:
                 # Need to apply this fee to only those which need reactivation
                 zt += RA_BURN_FEE
-                sum_burn_fees += RA_BURN_FEE
 
+                # Check here if payout amount is greater than, or equal to new zero threshold with reactivation fee added.
+                # If so, add burn fee to global total. If not, payout will not get appended to list and therefor burn fee should not be added to global total.
+                if pi.amount >= zt:
+                    sum_burn_fees += RA_BURN_FEE
+
+            # If payout total greater than, or equal to zero threshold, append payout record to master array
             if pi.amount >= zt:
                 payment_items.append(pi)
+            else:
+                logger.info("Skipping payout to {:s} ({:>10.6f} XTZ), reason: payout below minimum ({:>10.6f} XTZ)".format(pi.address, pi.amount / MUTEZ, zt / MUTEZ))
 
         if not payment_items:
             logger.info("No payment items found, returning...")
             return payment_items_in, 0, 0, 0
 
         # split payments into lists of MAX_TX_PER_BLOCK or less size
         # [list_of_size_MAX_TX_PER_BLOCK,list_of_size_MAX_TX_PER_BLOCK,list_of_size_MAX_TX_PER_BLOCK,...]
-        payment_items_chunks = [payment_items[i:i + MAX_TX_PER_BLOCK] for i in
-                                range(0, len(payment_items), MAX_TX_PER_BLOCK)]
+        payment_items_tz = [payment_item for payment_item in payment_items if payment_item.paymentaddress.startswith('tz')]
```

Comment:
It would be nice to have this as a function `split_payments()` so that it can be also unit tested.